### ISSUE TITLE
Purpose and goals of navigation areas

### ISSUE THREAD

**Issue by mikegianno** [transifex/txds#9]
At this point we have three navigation areas:

1. Header navigation area, the top level of navigation [generic and irrelevant to the displayed page]
2. Sub-header navigation area, right below the top navigation [relevant to the context of the specific page]
3. Side menu navigation, column navigation related to a top level navigation selection [sticky for a specific top level section]

This is a pretty cool information architecture that we should put some more reasoning than what is contained in the brackets ([]).

Additionally, it would be really really cool if we could answer the problem of where each type of navigation element should "live" on a navigation area.
Types of elements are:
- **User related elements** like User info, user settings, logout, notifications, messages
- **Org related elements** like Dashboard, Teams, Orders, Projects' list, create proj, search proj, system messages and verifications, org related notifications
- **System/Transifex related elements** like user account notifications, org account notifications, maintenance, usage help, changelog/what's new section

If we have a dedicated area for each type then we:
A. Have a more definite problem to solve when trying to add a new element on a navigation area
B. Our users will be already trained/acquainted with where should a "new thing" be placed.

**Comment by dontpanicgr** [comment]
Product Sitemap
![gloomap_33297581](https://user-images.githubusercontent.com/8539703/39748293-6450ff74-52b8-11e8-96dc-a28e8e0f6c6a.png)

The above is a section/page sitemap of our Information Architecture. Not included nodes.
_Nodes: A single node can be represented in a number of ways (a page, a pop-up, a light-box, a section of a page)


Further reading:
**What's a Sitemap?**
A site map is like a floor plan for your site.  Site maps give you a visual representation of the site's organization and how different sections are linked together.  

In short, a site map is a 30,000 foot view of the site (and all it's content) that puts the project team on the same page so everyone can get started doing their own high-level planning.
https://www.viget.com/articles/ux-101-the-site-map/
